mds/apis/agency_api/v0_x/vehicles.py

```diff
@@ -17,6 +17,7 @@
 
 
 class DeviceSerializer(serializers.Serializer):
+    """Expose devices as described in the spec."""
```

Comment:
Maybe specify what specs you are referring to

Reply:
This is the `agency API` package. :grin:

Reply:
I know but I think the phrasing would look more professional if you describe everything